src/GitHub.App/Services/GitHubContextService.cs

```diff
@@ -305,6 +307,55 @@ public bool TryOpenFile(string repositoryDir, GitHubContext context)
             }
         }
 
+        /// <inheritdoc/>
+        public string FindObjectishForTFSTempFile(string tempFile)
+        {
+            var match = tempFileObjectishRegex.Match(tempFile);
+            if (match.Success)
+            {
+                return match.Groups["objectish"].Value;
```

Comment:
Why did you name the group `objectish`?

Reply:
`(first 8 chars of a blob SHA)` ahh

Reply:
It started off being called blobish, but I stole the name from the `Lookup` method you found. 😄